lls/main.c

```diff
@@ -500,12 +531,12 @@ lls_stage2(void *arg)
 	 * Gatekeeper or Grantor.
 	 */
 
-	/*
-	 * TODO Have a different block set up RSS on the back interface,
-	 * and classify + distribute ND packets to the LLS block using
-	 * the DPDK libraries.
-	 */
-	if (lls_conf->nd_cache.iface_enabled(net_conf, &net_conf->back)) {
+	ret = register_nd_acl_rules(&net_conf->front);
```

Comment:
1. The comment before this change, which I can't comment directly, should be updated since we don't need to know from where the ND packets are coming from with the IPv6 filter. We should think of it as hardware filters.

2. Before installing ARP filters, the code test if ARP is enabled at that interface, why should ND do the same?